sqlglot/dialects/starrocks.py

```diff
@@ -31,6 +78,68 @@ class Parser(MySQL.Parser):
             "REGEXP": exp.RegexpLike.from_arg_list,
         }
 
+        PROPERTY_PARSERS = {
+            **MySQL.Parser.PROPERTY_PARSERS,
+            "DISTRIBUTED BY HASH": lambda self: self._parse_distributed_by("HASH"),
+            "DISTRIBUTED BY RANDOM": lambda self: self._parse_distributed_by("RANDOM"),
+            "PROPERTIES": lambda self: self._parse_wrapped_csv(self._parse_property),
+            "DUPLICATE KEY": lambda self: self._parse_duplicate(),
+            "PARTITION BY": lambda self: self._parse_partitioned_by(),
```

Comment:
Why are `PROPERTIES` and `PARTITION BY` here?

Reply:
`PARTITION BY` is unnecessary here, I deleted it.

`PROPERTIES` If you don't add it in PROPERTY_PARSERS, you will get an error:
```
'CREATE TABLE if not exists `sample_table` (
`tenantid` varchar(1048576) NULL COMMENT "",
`cr' contains unsupported syntax. Falling back to parsing as a 'Command'.
```

<Token token_type: TokenType.VAR, text: PROPERTIES, line: 18, col: 10, start: 734, end: 743, comments: []>
The error is `PROPERTIES (`. It seems that there is no `TBLPROPERTIES` in the hive code. This is strange 🤔

Reply:
Because `PROPERTIES` is also often used in doris's create table statement.
https://doris.apache.org/docs/sql-manual/sql-statements/Data-Definition-Statements/Create/CREATE-TABLE?_highlight=create&_highlight=table#properties

Reply:
`TBLPROPERTIES` is defined in sqlglot/parser.py:865, which allows hive to parse it well:
```
"TBLPROPERTIES": lambda self: self._parse_wrapped_properties(),
```
We are missing PROPERTIES, so I added `PROPERTIES` here

Reply:
create table `properties` docs links:
https://docs.starrocks.io/docs/sql-reference/sql-statements/data-definition/CREATE_TABLE/#properties

In addition, I did not put Properties into parse because it may affect other dialects. The following test will fail:
```
self.validate_identity(
"ALTER TABLE people SET PROPERTIES x = 'y'", check_command_warning=True
)
```